Source/BottomView/BottomContainerView.swift

```diff
@@ -51,7 +52,17 @@ class BottomContainerView: UIView {
     return configuration
     }()
 
+  lazy var topSeparator: UIView = { [unowned self] in
+    let view = UIView()
+    view.setTranslatesAutoresizingMaskIntoConstraints(false)
+    view.backgroundColor = self.configuration.backgroundColor
+    self.addSubview(view)
```

Comment:
Better to add subviews in ViewDidLoad, so the views would not be added ocassionally, when you just want to access them.